packages/discovery-react-components/src/components/DocumentPreview/components/SimpleDocument/__tests__/SimpleDocument.test.tsx

```diff
@@ -45,4 +47,41 @@ describe('SimpleDocument', () => {
       { exact: false }
     );
   });
+
+  it('renders a document with a different body field', () => {
+    const documentText = 'This is the text of the document.';
+    const customDoc = {
+      body_field: documentText
+    };
+    const context: Partial<SearchContextIFC> = {
+      componentSettings: {
+        fields_shown: {
+          body: {
+            field: 'body_field',
+            use_passage: true
+          }
+        }
+      }
+    };
+    const api: Partial<SearchApiIFC> = {};
+    let getByText: BoundFunction<GetByText> = () => {
+      throw new Error();
+    };
+
+    act(() => {
+      ({ getByText } = render(
+        wrapWithContext(
+          <SimpleDocument
+            document={customDoc}
+            setHideToolbarControls={(): void => {}}
+            setLoading={(): void => {}}
+          />,
+          api,
+          context
+        )
+      ));
+    });
+
+    getByText(documentText);
```

Comment:
this isn't really testing anything - can you add an `expect` here to validate something?

Reply:
The `get...` queries of `react-testing-library` do implicit assertions and will fail the test if they can't find the requested data. That lib also provides the same queries with `query...` instead -- these don't assert.

Reply:
So that throws an error if it fails, but does it show a success if it doesn't?

Reply:
Well, the test succeeds.